bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/entity/generic/BusinessPartnerDb.kt

```diff
@@ -104,6 +105,9 @@ class BusinessPartnerDb(
     @JoinColumn(name = "address_confidence_id", unique = true)
     var addressConfidence: ConfidenceCriteriaDb?,
 
+    @Column(name = "currentness")
+    var currentness: Instant? = null,
```

Comment:
@kunyao-cofinity-x, need to cross check on requirement as do we really need timestamp on output stage? Also even if we add then can you please streamline attribute name as `externalSequenceTimestamp` only as it will create confuse in future while reading DB table content.

Reply:
Hello @SujitMBRDI, thanks for looking into it. This attribute will be only applied to the input stage. I have pushed the modification of the column name.